client/extensions/zoninator/state/feeds/selectors.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+const getFeedsState = ( state ) => state.extensions.zoninator.feeds;
```

Comment:
It's more a guard for safety @ockham - why introduce something which can crash the entire app when it's easy to protect against. I've toyed with some kind of babel transform as well turning it into `(try { return state.extensions.zoninator.feeds } catch (e) { return {} })()` just for kicks and giggles but I didn't like that.

Too bad somebody hasn't created a way for some kind of computer program to read a source tree and point out places where we are using values which it can't prove actually exist at that point…